src/utils/getVisibleDays.js

```diff
@@ -21,10 +22,19 @@ export default function getVisibleDays(
 
     const currentDay = firstOfMonth.clone();
 
+    // create an array of week days based on the firstDayOfWeek
+    const mappedWeekDays = [];
+    for (let j = firstDayOfWeek; j < 7; j += 1) {
+      mappedWeekDays.push(j);
+    }
```

Comment:
```suggestion
    const mappedWeekDays = Array.from(
      { length: 7 - firstDayOfWeek },
      (_, i) => i + firstDayOfWeek),
    ).concat(Array.from({ length: 7 }, (_, i) => i));
```
rather than two for loops